libs/agno/agno/vectordb/qdrant/qdrant.py

```diff
@@ -233,10 +266,18 @@ def insert(self, documents: List[Document], filters: Optional[Dict[str, Any]] =
             document.embed(embedder=self.embedder)
             cleaned_content = document.content.replace("\x00", "\ufffd")
             doc_id = md5(cleaned_content.encode()).hexdigest()
+            vector = (
+                {
+                    "": document.embedding,
```

Comment:
`""` is the default dense vector name. See its creation at L160.
Reference: https://qdrant.tech/documentation/concepts/vectors/#named-vectors